ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -55,6 +56,18 @@ public class ArcticManagementConf {
           .defaultValue(3 * 60 * 1000L)
           .withDescription("Interval to refresh the external catalog.");
 
+  public static final ConfigOption<Integer> REFRESH_EXTERNAL_CATALOGS_THREAD_COUNT =
```

Comment:
We do better add these new configuration in `/dist/src/main/arctic-bin/conf/config.yaml`.